src/tests/query-entities-by-type-and-property-value.js

```diff
@@ -0,0 +1,81 @@
+import { SharedArray } from 'k6/data';
+import { createEntity } from '../api/create-entity.js';
+import { getEntities } from '../api/get-entities.js'
+import { uuidv4 } from 'https://jslib.k6.io/k6-utils/1.4.0/index.js';
+import { randomItem } from 'https://jslib.k6.io/k6-utils/1.2.0/index.js';
+import { randomIntBetween } from 'https://jslib.k6.io/k6-utils/1.2.0/index.js';
+import { randomString } from 'https://jslib.k6.io/k6-utils/1.2.0/index.js';
+
+export const options = {
+    vus: 10,
+    duration: '15m',
+    iterations: 100000,
+    thresholds: {
+        http_req_failed: ['rate<0.01'],
+        get_entities_duration: ['avg<1000', 'p(95)<1500']
+    },
+    setupTimeout: '5m',
+};
+
+const observedProperties = ['ammonium', 'waterTemperature', 'dissolvedOxygen'];
+const nonObservedProperties = ['name=="Name of my entity"', 'description=="A short description of my entity"', 'stationCode=="BSS002JAWM"'];
```

Comment:
find a better name for this because you have more than just a list of properties

Reply:
maybe `queryTermsOfNonObservedProperties` ?

Reply:
`nonObservedPropertiesStaticQueries`?

Reply:
good for me